src/lib/workspace/AllAnalyses.tsx

```diff
@@ -0,0 +1,423 @@
+                        else removePinnedAnalysis(data.row.analysis.id);
+                      }}
+                    />
+                  }
+                  label=""
+                />
+              </Tooltip>
+              <Link
+                to={Path.join(
+                  history.location.pathname,
+                  data.row.analysis.studyId,
+                  data.row.analysis.id
+                )}
+              >
+                {data.row.analysis.name}
+              </Link>
+            </>
+          ),
+        },
+        {
+          key: 'study',
+          name: 'Study',
+          sortable: true,
+          renderCell: (data: { row: AnalysisAndDataset }) => {
+            const { dataset } = data.row;
+            if (dataset == null) return 'Unknown study';
+            return (
+              <Link to={`${url}/${dataset.id[0].value}`}>
+                {dataset.displayName}
+              </Link>
+            );
+          },
+        },
+        {
+          key: 'created',
+          name: 'Created',
+          sortable: true,
+          renderCell: (data: { row: AnalysisAndDataset }) =>
+            new Date(data.row.analysis.created).toUTCString().slice(5),
+        },
+        {
+          key: 'modified',
+          name: 'Modified',
+          sortable: true,
+          renderCell: (data: { row: AnalysisAndDataset }) =>
+            new Date(data.row.analysis.modified).toUTCString().slice(5),
+        },
+      ],
+    }),
+    [
+      sortPinned,
+      filteredAnalysesAndDatasets,
+      tableSort,
+      selectedAnalyses,
+      pinnedAnalyses.length,
+      isPinnedAnalysis,
+      datasets?.records,
+      analyses,
+      deleteAnalyses,
+      removeUnpinned,
+      setSortPinned,
+      setTableSort,
+      history.location.pathname,
+      addPinnedAnalysis,
+      removePinnedAnalysis,
+      url,
+    ]
+  );
+  const theme = createMuiTheme(workspaceTheme);
+  return (
+    <ThemeProvider theme={theme}>
+      <div className={classes.root}>
+        <h1>My Analyses</h1>
+        {error && <ContentError>{error}</ContentError>}
+        {analyses && datasets && (
+          <Mesa.Mesa state={tableState}>
+            <div
+              style={{
+                display: 'flex',
+                alignItems: 'center',
+                gap: '1ex',
+              }}
+            >
+              <TextField
+                variant="outlined"
+                size="small"
+                label="Search analyses"
+                inputProps={{ size: 50 }}
+                value={searchText}
+                onChange={(e) => setSearchText(e.target.value)}
+              />
+              <span>
+                Showing {filteredAnalysesAndDatasets?.length} of{' '}
+                {analyses.length} analyses
+              </span>
+            </div>
+            {(loading || datasets == null) && <Loading />}
+          </Mesa.Mesa>
+        )}
+      </div>
```

Comment:
```suggestion
      <div className={classes.root}>
        <h1>My Analyses</h1>
        {error && <ContentError>{error}</ContentError>}
        {(loading || datasets == null) && <Loading />}
        {analyses && datasets && (
          <Mesa.Mesa state={tableState}>
            <div
              style={{
                display: 'flex',
                alignItems: 'center',
                gap: '1ex',
              }}
            >
              <TextField
                variant="outlined"
                size="small"
                label="Search analyses"
                inputProps={{ size: 50 }}
                value={searchText}
                onChange={(e) => setSearchText(e.target.value)}
              />
              <span>
                Showing {filteredAnalysesAndDatasets?.length} of{' '}
                {analyses.length} analyses
              </span>
            </div>
          </Mesa.Mesa>
        )}
      </div>
```

Reply:
Sorry, that's a much bigger-looking diff than it really is!

I moved the `<Loading />` line up to the top :-)